templates/atlus_fbn.bt

```diff
@@ -0,0 +1,143 @@
+typedef ubyte u8;
+typedef short s16;
+typedef ushort u16;
+typedef uint u32;
+typedef float f32;
+
+typedef enum<u16>
+{
+	Trigger = 4,
+	NPC = 5,
+} Major_Type;
+
+typedef struct
+{
+	u8 Unknown0;
+	u8 Unknown1;
+	u8 Unknown2;
+	u8 Unknown3;
+	Major_Type Major <name = "Major">;
+	s16 Minor <name = "Minor">;
+	u8 Res_ID <name = "Res ID">;
+	u8 Unknown;
+	u16 Field000;
+	float Field001 <name = "Collision Radius">;
+	float Field002;
+	float Field003;
+	float Field004;
+	float Field005;
+	float Field006 <name = "Unknown Float">;
+	float Field007;
+	float Field008;
+	float Field009;
+	float Field010;
+	float Field011 <name = "Unknown Float">;
+	float Field012;
+	float Field013;
+	float Field014;
+	float Field015;
+	float Field016 <name = "Unknown Float">;
+	float Field017;
+	float Field018 <name = "Y Pos">;
+	float Field019;
+	float Field020 <name = "X Pos">;
+	float Field021;
+	u32 Field022 <name = "Shadow Type">;
+	float Field023 <name = "Shadow Radius">;
+	u8 Unknown4;
+	u8 Unknown5;
+	u8 Unknown6;
+	u8 Unknown7;
+	u8 Unknown8;
+	u8 Unknown9;
+	u8 Unknown10;
+	u8 Unknown11;
+	if (Type_1_Entry_Size==320)
+	{
+		float Unknown12;
+		float Unknown13;
+		float Unknown14;
+		float Unknown15;
+		float Unknown16;
+		float Unknown17;
+		float Unknown18;
+		float Unknown19;
+		float Unknown20;
+		float Unknown21;
+		float Unknown22;
+		float Unknown23;
+		float Unknown24;
+		float Unknown25;
+		float Unknown26;
+		float Unknown27;
+		float Unknown28;
+		float Unknown29;
+		float Unknown30;
+		float Unknown31;
+		float Unknown32;
+		float Unknown33;
+		float Unknown34;
+		float Unknown35;
+		float Unknown36;
+		float Unknown37;
+		float Unknown38;
+		float Unknown39;
+		float Unknown40;
+		float Unknown41;
+		float Unknown42;
+		float Unknown43;
+		float Unknown44;
+		float Unknown45;
+		float Unknown46;
+		float Unknown47;
+		float Unknown48;
+		float Unknown49;
+		float Unknown50;
+		float Unknown51;
+		float Unknown52;
+		float Unknown53;
+		float Unknown54;
+		float Unknown55;
+		float Unknown56;
+		float Unknown57;
+		float Unknown58;
+		float Unknown59;
+		float Unknown60;
+		float Unknown61;
+		float Unknown62;
+		float Unknown63;
+	}
+} TFBNEntries1 <name = "FBNEntries">;
+
+typedef struct
+{
+	float Unknown0;
+	float Unknown1;
+	float Unknown2;
+	float Unknown3;
+	float Unknown4;
+	float Unknown5;
+	float Unknown6;
+	float Unknown7;
+	float Unknown8;
+	float Unknown9;
+	float Unknown10;
+	float Unknown11;
+	float Unknown12;
+	float Unknown13;
+} TFBNEntries2 <name = "FBNEntries">;
+
+typedef struct TableSegment
```

Comment:
This is not a good name for this struct as it's not represenative of its contents.

Reply:
Name should be FbnFile or something equivalent that represents the contents better